tasks/components.py

```diff
@@ -117,6 +117,7 @@ def has_type_component(content) -> bool:
     'comp/metadata/inventorychecks/inventorychecksimpl',
     'comp/metadata/inventoryhost/inventoryhostimpl',
     'comp/metadata/inventoryotel/inventoryotelimpl',
+    'comp/metadata/haagent/haagentimpl',
```

Comment:
The comment 40 lines aboves asks for a specific component "style" (package organization), any chance you can update your component to match that ? (and thus not require adding the exception here)

Reply:
@pgimalac ah yes, sorry, I actually already update the component to match the new style previously.

just removed the exception in `tasks/components.py` 

see d1e127fc2e

Reply:
Thanks haha, sorry I didn't check the implementation before asking 😄